docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst

```diff
@@ -100,10 +100,137 @@ audience.
+CVD rates are changing differently might lead us to make a better forecast for 
+the future. So, it seems relevant to include race here.  
+
+Second, let’s think about if this will lead to changes in policies. Here again, 
+it seems plausible that if we see a sharper increase  in CVD incidence among some 
+racial groups, decision-makers might deploy racially aware intervention strategies. 
+So here, including something about race, ethnicity, and/or ancestry seems relevant. 
+
+Last, let’s think through the possible causal pathways and decide what metric(s) 
+we want to include. In the below diagrams, the solid lines indicate casual pathways, 
+while the dotted lines indicate correlation. 
+
+**Option 1a: Race** 
+
+.. image:: race_a_v1.svg
+
+We stated above that its commonly known that CVD rates vary with race. This is 
+well documented and true. It's easy therefore to think about the casual diagram 
+as a single line between race and disease. But all we know is that there is a 
+correlation between race and CVD. We want to more closely examine what might be 
+causing this correlation. 
+
+Given our defintion of race as physical characteristics, for race to cause the 
+disease, it means that the physical characteristics are causing the disease itself. 
+There might be a few cases where this causal pathway is correct, but they are limited 
+in scope. For example, lighter skinned individuals tend to get skin cancer at higher 
+rates. In this case, the physical characteristic of race impacts the disease. However, 
+this is quite uncommon. 
+
+There isn't any known reason why skin color or other physical characteristics are 
+tied to CVD, so we can dismiss this option for our research question. 
+
+**Option 1b: Race** 
+
+.. image:: race_b_v1.svg
+
+A much more common option is race “leads” to experiences of racism which in turn leads 
+to the disease. We put “leads” in quotes as we don’t want to imply that existing with 
+certain physical characteristics “causes” racism, but rather that racism is the result 
+of years of history defining groups based on physical characteristics. 
+
+This casual pathway is much more common, and seems more applicable to our example with CVD. 
+However, considering the exact pathway here is important. We list a few possible pathways for 
+CVD based on race. Note that these do overlap with ethnicity in some cases. This is 
+because discrimination can impact culture/ethnic practices. These are often 
+rooted in racism. 
+
+- Historical red lining leading to African Americans living in food deserts, having less available green space, or living in neighborhoods with fewer social services
+- Historical racism leading to lower wealth in some communities, reduced access to health insurance, or limited access to quality medical care
+- Microaggressions experienced by non-white individuals leading to elevated blood pressure 
+- Racism in healthcare settings leading to delayed treatment once a person seeks care 
```

Comment:
I think these are really helpful examples! I like how they span a wide range of time and scale.